app/views/users/show.html.erb

```diff
@@ -1,3 +1,30 @@
-<h1>Here is a page for showing user details </h1>
+<div class="user_con">
+  <div class="user_img">
+    <%= image_tag("image5.png", width:'150px', height:'200px') %>
+  </div>
+  <div class="user_data">
+    <h2><%= @user.name%></h2>
+    <p>Number of posts: <%= @user.posts_counter%></p>
+  </div>
+</div>
 
-<p>This page is under construction</p>
+<div class="user_con2">
+  <div class="user_data2">
+    <h2>Bio</h2>
+    <p class="bio"><%= @user.bio%></p>
+  </div>
+</div>
+
+<% @user.recent_3_posts.each do |post| %>
+  <div class="comment_con">
+      <h2><%= post.title%></h2>
+      <p><%= post.text%></p>
+      <h3>Comment <%= post.comments_counter%>,  <span>Likes <%= post.likes_counter%></span></h3>
+  </div>
+<%end%>
```

Comment:
- [ ]  `recent_post` method is supposed to render the user's 3 most recent post. However Im getting an error:
<img width="653" alt="Screenshot 2022-05-03 at 12 41 26" src="https://user-images.githubusercontent.com/77038610/166425878-c809fe5f-4fc7-4a36-b889-22ee3bb1204b.png">
 As you can guess by now, this error occurred, because `recent_3_posts` method does not exit in users model. Instead, it has recent_post.
Please make sure that users show page displays recent 3 posts successfully.